src/components/specialist_page/specialist_page.js

```diff
@@ -18,8 +18,8 @@ const SpecialistPage = () => {
         "bio",
         "rating"
     ]
-
-    const url = "https://g6bcybbjx1.execute-api.eu-central-1.amazonaws.com/"
+    
+    const url = "http://3.65.253.196/"
```

Comment:
Didn't we use .env file for storing basic url?

Reply:
Yeah, I fixed it.